assets/price-per-item.js

```diff
@@ -0,0 +1,96 @@
+class PricePerItem extends HTMLElement {
+  constructor() {
+    super();
+    this.input = this.querySelector('input');
```

Comment:
Which input are we trying to query for here? I would try to query for something less generic. I think you could add a unique id to the input in Liquid and use that to query using `document.querySelector`

Reply:
The input already has `id="Quantity-{{ section.id }}"` I added `data-section-id="{{ section.id }}"` to the component and now I can query it `this.input = document.getElementById(`Quantity-${this.dataset.sectionId}`);`